superset/assets/javascripts/explorev2/exploreUtils.js

```diff
@@ -1,26 +1,40 @@
 /* eslint camelcase: 0 */
-export function getExploreUrl(form_data, endpoint = 'base', force = false) {
+import URI from 'urijs';
+
+export function getExploreUrl(form_data, endpointType = 'base', force = false, curUrl = null) {
   if (!form_data.datasource) {
     return null;
   }
+
+  let uri = URI(window.location.search);
+  // curUrl is used for tests
```

Comment:
could we pass in search params to this function rather than using `window.location.search`, then we wouldn't need this block for tests.